packages/ui/components/Pages/Program/TableRow.tsx

```diff
@@ -12,43 +13,101 @@ type TableRowProps = {
 const TableRow = ({ item }: TableRowProps) => {
   const [task, setTask] = useState(item.name);
   const [budget, setBudget] = useState(item.budget);
-  const [duration, setDuration] = useState(item.duration);
+  const [showDropdown, setshowDropdown] = useState(false);
   const [selectedStatus, setStatus] = useState(item.status);
   const [isOwnerDetailsVisible, setOwnerDetailsVisible] = useState(
     false,
   );
   const [startDate, setStartDate] = useState(item.startDate);
   const [endDate, setEndDate] = useState(item.endDate);
+  const [hover, setHover] = useState(false);
+
+  let statusClass = 'status-not-started';
+  if (selectedStatus === 'completed') {
+    statusClass = 'status-completed';
+  }
+
+  if (selectedStatus === 'in-progress') {
+    statusClass = 'status-in-progress';
+  }
+  if (selectedStatus === 'not-started') {
+    statusClass = 'status-not-started';
```

Comment:
@pipesort123TP  why do we have to prefix like this one, can we remove `status-` prefix please?